rootfs/Dockerfile

```diff
@@ -36,6 +36,8 @@ RUN mkdir -p /usr/share/capch/images && \
         skopeo copy docker://$image oci-archive:/usr/share/capch/images/$(echo $image | sed -r 's/[/:]/_/g').tar:$image; \
     done
 
+RUN  update-alternatives --set iptables /usr/sbin/iptables-legacy && update-alternatives --set ip6tables /usr/sbin/ip6tables-legacy
```

Comment:
Better add a wrap between two bash commands

Reply:
https://github.com/everoute/everoute/blob/main/build/script/iptables-wrapper
FYI, fork from kubernetes,

Reply:
What does the word `wrap` mean here, a bash command that can switch between `iptables-legacy` and `iptables-nft` automatically?

But the purpose of this patch is switching to `iptables-legacy` explicitly in rootfs, since there are some required modules missing from our linux kernel config in nf_tables, like contrack, masquerade and etc. Otherwise, we have to modify the kernel config file, maybe not a good idea.

Reply:
Sorry, I mean a line wrap